src/main/java/org/apereo/openlrs/exceptions/caliper/InvalidCaliperFormatException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2015 Unicon (R) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ * http://www.osedu.org/licenses/ECL-2.0
+
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+package org.apereo.openlrs.exceptions.caliper;
+
+/**
+ * @author Lance E Sloan (lsloan at umich dot edu)
+ */
+public class InvalidCaliperFormatException extends RuntimeException {
```

Comment:
Yes, good point.  I was following the example of the xAPI code, but I like your idea better.